src/hocs/withWeb3.js

```diff
@@ -0,0 +1,22 @@
+import React, {Component} from "react";
+import PropTypes from "prop-types";
+
+const withWeb3 = mapWeb3ToProps => WrappedComponent =>
```

Comment:
Same comment about using a default value for this argument and throwing an error if it's the wrong type.